src/program/media/filesystem_entry.py

```
@@ -64,96 +59,39 @@ class FilesystemEntry(db.Model):
         lazy="selectin"
     )
 
+    __mapper_args__ = {
+        "polymorphic_identity": "base",
+        "polymorphic_on": "entry_type",
+    }
```

Ruff (0.13.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)